eth-reference-tests/src/test/java/pegasys/artemis/reference/TestSuite.java

```diff
@@ -16,6 +16,10 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import com.google.errorprone.annotations.MustBeClosed;
+import kotlin.Pair;
```

Comment:
what is this for?  curious

Reply:
is it a tuple?

Reply:
Yes, it is the kotlin library for Tuple which is JVM compatible.